spec/features/users_can_view_programme_level_activities_spec.rb

```diff
@@ -1,20 +1,26 @@
 RSpec.feature "Users can view programme level activities" do
   context "when the user belongs to BEIS" do
     let(:user) { create(:beis_user) }
+    let(:fund_activity) { create(:fund_activity, organisation: user.organisation) }
```

Comment:
This was already in there, but do we need to create this fund here? It doesn't seem to be used beyond setting the programme's parent, so I'd have thought we could just leave it as the default random parent. Same in the other context